vpr/src/route/partition_tree.h

```diff
@@ -83,9 +86,14 @@ class PartitionTree {
     /** Access root. Shouldn't cause a segfault, because PartitionTree constructor always makes a _root */
     inline PartitionTreeNode& root(void) { return *_root; }
 
+    void update_nets(const std::vector<ParentNetId>& nets);
```

Comment:
Doxygen comment. Explain that it looks at the bounding box and moves the net up or down the partition tree; needed because the router can adjust bounding boxes.